packages/docusaurus/src/server/configValidation.ts

```diff
@@ -233,6 +233,9 @@ export const ConfigSchema = Joi.object<DocusaurusConfig>({
       .try(Joi.string().equal('babel'), Joi.function())
       .optional(),
   }).optional(),
+  markdown: Joi.object({
+    mermaid: Joi.boolean().optional(),
+  }).optional(),
```

Comment:
_Sigh_ We don't have a better way to do this, but ideally it shouldn't be the core to validate this option...